templates/blocks/review_item.liquid

```diff
@@ -0,0 +1,23 @@
+<div class="sc-review">
+  <div class="sc-review-wrapper sc-flex-col sc-justify-center sc-align-items-center" id="{{ content_block.identifier }}">
+    <div class="sc-review-rating">
+      {% render 'shared/icons/star', style_class: 'sc-icon-star' %}
+      {% render 'shared/icons/star', style_class: 'sc-icon-star' %}
+      {% render 'shared/icons/star', style_class: 'sc-icon-star' %}
+      {% render 'shared/icons/star', style_class: 'sc-icon-star' %}
+      {% render 'shared/icons/star', style_class: 'sc-icon-star' %}
+    </div>
+
+    {%- if content_block.title != blank %}
+      <p class="sc-review-message sc-m-none sc-font-medium">
+        {{ content_block.title }}
+      </p>
+    {%- endif %}
+
+    {%- if content_block.subtitle != blank %}
+      <p class="sc-review-content sc-m-none sc-font-medium sc-font-bold">
```

Comment:
`-content` seems to be an element part of the review structure and not a modifier.